.travis.yml

```diff
@@ -19,7 +19,7 @@ before_install:
   - openssl aes-256-cbc -K "$encrypted_5ebd3ff04788_key" -iv "$encrypted_5ebd3ff04788_iv" -in src/bin/travis/resources/jesConf.tar.enc -out jesConf.tar -d || true
 env:
   global:
-    - CENTAUR_BRANCH=develop
+    - CENTAUR_BRANCH=4648da44fc1695d5306e9b4f05e96fa5a5cadb8f
```

Comment:
This commit was the last commit merged to Centaur before we released 25. Just waiting for the tests to pass on this commit and will update it to a proper branch if it turns out to be correct.